app/javascript/components/posts/PostsIndex/Post.module.scss

```diff
@@ -0,0 +1,51 @@
+.lightContainer {
+  width: 900px;
+  margin: auto;
+  background-color: #fff;
+  padding: 40px;
+  margin: 10px;
+  border-radius: 8px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.postSubBox {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  
+}
+
+.subBoxImg {
+  max-width: 300px;
+  height: 150px;
+  margin-left: 20px;
+
+}
+
+.title {
+  text-align: left !important;
+}
+
+p {
```

Comment:
We can have problems with this style.  this style will be applied for all p. May create custom class